test/kubetest/pods/vppagent_dataplane.go

```diff
@@ -1,3 +1,4 @@
+//nolint
```

Comment:
Hmm, why?

Reply:
Linter justification for code repetition

Reply:
I think all //nolint should be removed before merging. Thanks.